_includes/header.html

```diff
@@ -5,14 +5,15 @@
             <p class="intro__post">{{ site.location }}</p>
         </div>
 
-        {% if site.github_username or site.codepen_username or site.twitter_username or site.email_address %}
         <div class="social">
+<a href="http://micca.report/index.html">Home</a> 
+<a href="http://micca.report/validate.html">Validate your data</a> 
+<a href="http://micca.report/resources.html">Resources</a> 
```

Comment:
These links should be relative since they're for relative resources.  That will avoid the http vs https problem with hard coding like this.  Also these links won't point to the local copy of the resources.

I'll fixup after the merge.